js/hoverzoom.js

```diff
@@ -1681,7 +1681,7 @@ var hoverZoom = {
                     imgThumb = hz.currentLink.find('[style]');
                     if (imgThumb.length > 0) {
                         imgThumb = $(imgThumb[0]);
-                        lowResSrc = hz.getThumbUrl(imgThumb);
+                        lowResSrc = hz.getThumbUrl(imgThumb[0]);
```

Comment:
@GrosPoulet Does this work correctly? Line 1683 takes the first element of the list. But then line 1684 does it again? Was it a nested list?

Reply:
@extesy function `getThumbUrl(arg)` takes a DOM element as argument, not a JQuery object.